src/main/java/de/samply/share/broker/utils/connector/IcingaConnector.java

```diff
@@ -193,11 +193,8 @@ private static void sendPerformanceReport(String sitename, StatusReportItem stat
       boolean isRetry) throws IcingaConnectorException {
     try {
       HttpPost httpPost;
-      if (ProjectInfo.INSTANCE.getProjectName().toLowerCase().equals("samply")) {
-        httpPost = createPost(sitename, statusReportItem.getParameterName() + "-gba");
-      } else {
-        httpPost = createPost(sitename, statusReportItem.getParameterName());
-      }
+      httpPost = createPost(sitename, statusReportItem.getParameterName()
+          + project);
```

Comment:
shouldn't we pass the minus sign here? or should the project start with a minus? I assume the project should be just called `gba`

Reply:
ok